src/directives/for-of/for_of.directive.ts

```diff
@@ -224,6 +224,17 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         if (this.igxForRemote) {
             return;
         }
+        if (endingIndex === this.igxForOf.length) {
+            // scrolled to bottom
+            const h = this.dc.instance._viewContainer.element.nativeElement.clientHeight;
+            const diffContent = this.state.chunkSize * parseInt(this.igxForItemSize, 10) - parseInt(this.igxForContainerSize, 10);
+            const totalDiff = h - parseInt(this.igxForContainerSize, 10) - diffContent;
```

Comment:
To calculate `totalDiff` there is addition and subtraction of one and the same parameter `igxForContainerSize`. It does not seem to take account for the final result of `totalDiff`. It is the same as(no need of `diffContent`):
`totalDiff = h - this.state.chunkSize * parseInt(this.igxForItemSize, 10);`
@MayaKirova

Reply:
Yeah. Makes sense.